app/api/tests/test_api.py

```diff
@@ -1418,6 +1419,12 @@ def test_give_invalid_data_to_conll_parser(self):
             self.parser_helper(filename='labeling.invalid.conll',
                                parser=CoNLLParser())
 
+    def test_give_classification_data_to_fasttext_parser(self):
+        self.parser_helper(filename='file.fast', parser=FastTextParser())
+
+    def test_give_seq2seq_data_to_fasttext_parser(self):
```

Comment:
FastText does not support seq2seq and this test does the same thing as the classification one.